gender_novels/analysis/visualizations/datagraphs_functions.py

```diff
@@ -2,58 +2,87 @@
 import seaborn as sns
 from gender_novels.corpus import Corpus
 
-def plt_pubyears(years):
+def plt_pubyears(years,name_of_data):
```

Comment:
For this and the following functions: I think I'd call the param `corpus_name` instead of `name_of_data` for clarity.
Also, possible future modifications: We might want to two parameters. A `corpus name` (e.g. `sample_novels`) for the file name and a `display_name` (e.g. `Sample Novels`) for the title displayed in the chart. That modification is not very urgent, though.